zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/deployment/model/element/ExecutableCallActivity.java

```diff
@@ -15,6 +15,7 @@ public class ExecutableCallActivity extends ExecutableActivity {
 
   private boolean propagateAllChildVariablesEnabled;
   private boolean propagateAllParentVariablesEnabled;
+  private int lexicographicIndex;
```

Comment:
🔧 Let's add JavaDoc clarifying that this concerns the lexicographic index of call activities within the process it is part of.